damnit/gui/plot.py

```diff
@@ -279,8 +279,8 @@ def update_canvas(self, xs=None, ys=None, image=None, legend=None, series_names=
                 self._axis.set_xlabel("")
                 self._axis.set_ylabel("")
             else:
-                vmin = np.nanquantile(image, 0.01, interpolation='nearest')
-                vmax = np.nanquantile(image, 0.99, interpolation='nearest')
+                vmin = np.nanquantile(image, 0.01, method='nearest')
+                vmax = np.nanquantile(image, 0.99, method='nearest')
```

Comment:
There was a weird compat issue with this that made me not fix it earlier, but now I don't remember what it was... let's do it anyway, if it breaks something we can fix it later.

Reply:
There's some similar code in ctxrunner which doesn't pass method/interpolation at all - maybe some target Python for running the context file had an older version of matplotlib and that bit you at some point? That's just a guess, though.

Reply:
Yes, maybe that was it :thinking: